customizations/README.md

```diff
@@ -236,6 +239,12 @@ Add the following lines to you `customizations.yaml` to connect to the external
     port: "6379"
     existingSecret: "mycarto-custom-redis-secret"
     existingSecretPasswordKey: "password"
+    tls_enabled: true
+    # If your Redis TLS Cert it's self-signed
```

Comment:
could we add a more explicit comment that only apply if it is a self-signed? e.g: only applies if...